torchtitan/datasets/multimodal/clip.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Any, List, Mapping, Optional, Tuple
+
+import torch
+
+import torchvision
```

Comment:
This is just a quick scan for this PR. I need more time to look into this PR. But we should add torchvision as the dependent of Torchtitan, if haven't.

Reply:
Yes! Don't worry, it will take time 😅